### DISCUSSION TITLE
PWA - error "could not complete the operation (not enough memory)" - how to solve?

### DISCUSSION THREAD

**Discussion by zorgo** [quasarframework/quasar#11474] [CLI - PWA mode]
Hi!  Please help me - I have a big trouble...
There is a simple PWA application developed for sending reports on the job performed - the report form is in the application

On some phones (I saw on Android) the error "could not complete the operation (not enough memory)" appears when adding a photo from the camera - it is enough to take one - maximum two photos.
I read on the forum that it is necessary to check disabling "background Activity preemption" in the developer's parameters - everything is off - however, there is an error and is apparently related to memory - because when adding pictures from the gallery - the application does not crash - but adds a maximum of 2-3 pictures.

**Comment by metalsadman** [comment]
try this https://forums.oneplus.com/threads/low-memory-during-pic-upload.107187/#post-11204419
or build a debug mode, then check for the debugger message, or provide a repo so peeps can test.
